flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -125,6 +126,10 @@ Actor with ActorLogMessages with ActorLogging {
   override def postStop(): Unit = {
     log.info(s"Stopping job manager ${self.path}.")
 
+    // disconnect the registered task managers
+    instanceManager.getAllRegisteredInstances.asScala.foreach{
+      _.getTaskManager ! Disconnected("JobManager is stopping")}
+
     for((e,_) <- currentJobs.values){
       e.fail(new Exception("The JobManager is shutting down."))
```

Comment:
Since we are cleaning up messages, maybe remove "The" so it is consistent with other messages.